internal/evaluator/builder.go

```diff
@@ -38,6 +38,12 @@ type Builder struct {
 	Load        func(desc protoreflect.MessageDescriptor) MessageEvaluator
 }
 
+type StandardConstraintResolver interface {
```

Comment:
This was from `internal/evaluator/resolver.go`, but the rest of that file is moved out of `internal`.